api/proto/config/devmodel.proto

```diff
@@ -64,15 +70,20 @@ message SystemAdapter {
         // deprecate: have a separate device policy object in the API
         bool uplink = 3;
 
+        // XXX Note that this is the static information from the model.
+        // Current configuration is in systemAdapter
+        PhyIoMemberUsage usage    = 6;
+        PhyIOUsagePolicy usagePolicy    = 7;
+
         // attach this network config for this adapter
         string networkUUID = 4;
 
         // if its static network we need ip address
         string addr = 5;
 
-        // alias/logical name which will be reported to zedcloud
-        // and used for app instances
-        string logicalName = 6;
+        // physicalLabel / LogicalLabel - Point to the corresponding PhysicalIo
+        string logicalLabel = 6;
+        string physicalLabel = 7;
```

Comment:
If the name is set to the physical label (based on the comment you added), why do you need to add a separate physicalLabel field?

Reply:
Like the way we use names like "uplink"/"freeuplink" for grouping all management ports, should we also enable group labels for grouping multiple logical/physical ports under a given grouping label? For ex: Group multiple app-shared ports and attach to a local network instance.

Reply:
I thought I would deprecate name and delete it later on.. Calling it physicalLabel in the log run is more clearer..

Reply:
Please ignore this change.. The config file - I put them in a separate PR:
https://github.com/lf-edge/eve/pull/355/files